src/qml/patchmanager.cpp

```diff
@@ -115,9 +115,7 @@ void PatchManager::restartServices()
     }
 
     if (m_homescreenNeedRestart) {
-        QStringList arguments;
-        arguments << "--user" << "restart" << "lipstick.service";
-        QProcess::startDetached("systemctl", arguments);
+        restartLipstick();
```

Comment:
You can also remove the second if (m_homescreenNeedRestart) that is following this if